NebulaClient/GameLogic/PlayerManager.cs

```diff
@@ -8,6 +8,7 @@ public class PlayerManager
         Dictionary<ushort, Player> remotePlayers;
         Dictionary<ushort, RemotePlayerModel> remotePlayerModels;
 
+        public bool WeAreMainPlayer { get; set; } = false;
```

Comment:
Should rename to `IsMasterClient` and you can omit the `= false;` since the value is false by default.